CHANGES.md

```diff
@@ -3,6 +3,11 @@
 * Setting a dataset's `BoundingBox` in the server configuration 
   is now recognised when requesting the dataset details. (#845)
 
+* It is now possible to enforce the order of variables reported by 
+  xcube server. The new server configuration key `Variables` can be added 
+  to dataset configurations. Is a list of wildcard patterns that 
+  determine the variables' order. (#835) 
```

Comment:
Please add, that if for a dataset only a subset of variables is defined and **not** followed by `- "*"`, then only the specified variables are displayed.

Reply:
Good point!